### ISSUE TITLE
Syntax error in > 1.2.0

### ISSUE THREAD

**Issue by rteshnizi** [jcubic/jquery.terminal#308]
I get a syntax error when building my code.
```
SyntaxError: Cannot declare a parameter named 'request' in strict mode
  at /bower_components/jquery.terminal/js/jquery.terminal.js:3310
```

The same error is thrown with the minified file as well.

**Comment by jcubic** [comment]
Fixed in version 1.4.1 with npm I'll wait until you test.

Where do you see this error, (what browser/os)? I don't see it in chrome and lint didn't report it. Maybe this should reported to eslint (or maybe I don't have rule for that).

**Comment by rteshnizi** [comment]
yep it is fixed.

I use jshint and jscs. But this error is caught by PhantomJs. I use PhantomJs 2.1.1.
